prod-devel-rcar.yaml

```diff
@@ -329,6 +333,7 @@ parameters:
   MACHINE:
     desc: "RCAR Gen3-based device"
     salvator-x-m3:
+      default: true
```

Comment:
Looks like spurious change

Reply:
Seems like PR is not rebased on HEAD. This line was removed by PR merged one day ago.

Reply:
Fixed.